templates/store/store.html

```diff
@@ -31,59 +31,55 @@ <h2>Big data</h2>
   </div>
 </div>
 
-<div class="p-strip is-shallow u-no-padding--bottom">
+<div class="p-strip--accent is-slanted">
   <div class="row">
-    <div class="u-equal-height">
-      <div class="col-6 p-card">
+    <div class="col-6">
       <div class="col-2 u-align--center">
-        <img src="https://assets.ubuntu.com/v1/96c61b0d-charm-icon.png" alt="Illustration of a Charm pictogram">
+        <img src="https://assets.ubuntu.com/v1/96c61b0d-charm-icon.png" alt="">
       </div>
       <div class="col-4 u-no-margin--left">
+        <h4 class="p-muted-heading">Charms</h4>
         <p>Charms are sets of scripts that simplify the deployment and management tasks of a service. They are regularly reviewed and updated.</p>
         <a href="{{ url_for('jaasstore.search', type='charm') }}" class="p-button--neutral">View all the charms</a>
       </div>
     </div>
-      <div class="col-6 p-card">
+      <div class="col-6">
       <div class="col-2 u-align--center">
-        <img src="https://assets.ubuntu.com/v1/d4ffa9bc-bundle-icon.png" alt="Illustration of a Bundle pictogram">
+        <img src="https://assets.ubuntu.com/v1/d4ffa9bc-bundle-icon.png" alt="">
       </div>
       <div class="col-4 u-no-margin--left">
+        <h4 class="p-muted-heading">Bundles</h4>
         <p>Bundles are collections of charms that link applications together, so you can deploy whole chunks of infrastructure in one go.</p>
         <a href="{{ url_for('jaasstore.search', type='bundle') }}" class="p-button--neutral">View all the bundles</a>
       </div>
     </div>
-    </div>
   </div>
 </div>
 
-<div class="p-strip is-shallow is-bordered">
+<div class="p-strip is-bordered">
   <div class="row">
     <div class="col-12">
-      <h3 class="p-muted-heading u-align--center" style="max-width:100%">Categories</h3>
+      <h3 class="p-muted-heading">Categories</h3>
+      <p class="u-no-max-width u-no-margin--bottom">
+        <a class="p-link--strong" href="{{ url_for('jaasstore.search', q='databases') }}">databases</a>,
```

Comment:
This categories 'list' is very hard to read and to see the delineation between the items. 

These are bolded here but not in the design, although it's no easier to read in the design.

Reply:
Good catch, updated them to soft links.